src/main/java/seedu/address/logic/parser/CliSyntax.java

```diff
@@ -12,5 +12,6 @@ public class CliSyntax {
     public static final Prefix PREFIX_ADDRESS = new Prefix("a/");
     public static final Prefix PREFIX_TAG = new Prefix("t/");
     public static final Prefix PREFIX_DATETIME = new Prefix("d/");
+    public static final Prefix PREFIX_IMPORTANT_DATE = new Prefix("e/");
```

Comment:
it might be confusing for user to have adding of ID be n/ d/ 
then introduce deleting to use e/